astropy/time/core.py

```diff
@@ -1082,7 +1082,8 @@ def _apply(self, method, *args, format=None, cls=None, **kwargs):
 
             setattr(tm, attr, val)
 
-        # Copy other 'info' attr only if it has actually been defined.
+        # Copy other 'info' attr only if it has actually been defined and the
```

Comment:
This can now be removed/reverted? Or link to this PR perhaps instead of the issue.